includes/class-coblocks-block-assets.php

```diff
@@ -295,7 +296,7 @@ public function frontend_scripts() {
 		$vendors_dir = CoBlocks()->asset_source( 'js', 'vendors' );
 
 		// Masonry block.
-		if ( has_block( 'coblocks/gallery-masonry' ) || has_block( 'core/block' ) ) {
+		if ( is_admin() || has_block( 'coblocks/gallery-masonry' ) || has_block( 'core/block' ) ) {
```

Comment:
Is there a check for a Gutenberg admin page instead?

Reply:
Yes, we actually have `is_page_gutenberg()` we can use. Switching to it in the next push.